lib/plugins.js

```diff
@@ -352,19 +352,19 @@ function plugins (settings, deviceId) {
 
     const phone = tx.phone
     const timestamp = dateFormat(new Date(), 'UTC:HH:MM Z')
-    const rec = {
-      sms: {
-        toNumber: phone,
-        body: `Your cash is waiting! Go to the Cryptomat and press Redeem within 24 hours. [${timestamp}]`
-      }
-    }
-
-    return sms.sendMessage(settings, rec)
-      .then(() => {
-        const sql = 'update cash_out_txs set notified=$1 where id=$2'
-        const values = [true, tx.id]
+    return sms.getCashOutReadySms(phone, timestamp)
+      .then(msg => {
+        const rec = {
+          sms: msg
+        }
```

Comment:
Maybe we should change 'msg' to something like 'smsStructure' since this is an object, what do you think?